servers/rendering/renderer_rd/renderer_scene_render_rd.cpp

```diff
@@ -1083,43 +1085,62 @@ int RendererSceneRenderRD::get_directional_light_shadow_size(RID p_light_intance
 
 //////////////////////////////////////////////////
 
-RID RendererSceneRenderRD::camera_effects_allocate() {
-	return camera_effects_owner.allocate_rid();
+RID RendererSceneRenderRD::camera_attributes_allocate() {
```

Comment:
Should we move this into one of our storage objects? Or create a Camera attributes storage object?

Reply:
I created a CameraAttributes storage object that is shared by the various backends